index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <tile></tile>
+        <link href="style.css" rel="stylesheet">
+    </head>
+
+    <body> 
+        <header>
+          <nav>  
+            <div id="topIcons">

```

Comment:
- [ ] You've made a polite request here, but your suggestion lacks a `why`. Include an explanation of why you think the student is `overusing` tags. 
Please always include an explanation, a "why", for each requested change. For example:
- This is a well-explained request: 
  Why don't you use an article tag for the .first-article element instead of a div, improving the HTML structure and making the page more accessible? 
- This request lacks a "why": 
Why don't you use an article tag here instead of a div?